modules/package-config.test.js

```diff
@@ -10,7 +10,7 @@ jest.mock("find-package-json");
 // #region Imports
 // -----------------------------------------------------------------------------------------
 
-const { CLI_NAME } = require("./constants");
+const { BIN, CLI_NAME, PACKAGE_JSON, ENTRYPOINT } = require("./constants");
```

Comment:
Excellent job refactoring to bring these in from constants. Overall excellent job adding these tests!